src/os/bluestore/BlueFS.cc

```diff
@@ -2543,7 +2543,6 @@ void BlueFS::_compact_log_sync_LNF_LD()
     prefer_bdev,
     0,
     super.memorized_layout);
-  logger->inc(l_bluefs_log_compactions);
```

Comment:
Would it be better to remove this line from _rewrite_log_and_layout_sync_LNF_LD? In this case _compact_log_sync_LNF_LD & _compact_log_async_LD_LNF_D will be in sync in this regard...

Reply:
Thanks for your comment. We can see that `_rewrite_log_and_layout_sync_LNF_LD` is also called in other places. Will modifying `_rewrite_log_and_layout_sync_LNF_LD` reduce the count of `l_bluefs_log_compactions`?

Reply:
Other calls of _rewrite_log_and_layout_sync_LNF_LD() are from API bound to ceph-bluestore-tool exclusively. Which generally doesn't care about perf counters. So I think it's safe to go this way.

Reply:
You are right! I have remove this line from `_rewrite_log_and_layout_sync_LNF_LD`.